app/scripts/lib/getObjStructure.js

```diff
@@ -6,9 +6,14 @@ import { cloneDeep } from 'lodash';
 // {
 //   "data": {
 //     "CurrencyController": {
-//       "conversionDate": "number",
-//       "conversionRate": "number",
 //       "currentCurrency": "string"
+//       "currencyRates": {
+//         "ETH": {
+//           "conversionDate": "number",
+//           "conversionRate": "number",
+//           "usdConversionRate": "number",
+//         }
+//       },
```

Comment:
🤔  probably don't need this?

Reply:
I think this is helpful to have here. Personally, it isn't immediately obvious what the functions in this file do on first glance